src/utils/typeormSwitch.ts

```diff
@@ -0,0 +1,9 @@
+import * as ProdTORM from 'typeorm' // for prod
+
+let TORM = ProdTORM
+
+if (process.env.NESTJS_ADMIN_TYPEORM_PATH) {
+  TORM = require(process.env.NESTJS_ADMIN_TYPEORM_PATH)
+}
+
+export = TORM
```

Comment:
`typeormSwitch` isn't a great name. Maybe `typeormProxy`? And we need a big fat comment at the top of this file to explain why we're doing this